docs/benchmark_suite/README.md

```diff
@@ -159,27 +159,27 @@ A typical status quo, not collecting any provenance data and not using workflows
 System-level, workflow/language-level, and application-level are on a curve, increasing cost and enabling more provenance applications.
 
 The initial jump in cost from nothing to system-level is low because the user need not change _anything_ about their application;
-  they merely need to install some provenance tracer onto their system and run their code, without modifying it, in the tracer. ^[DSK: what about the performance penalty? Since you talk about performance in contributions, I think you have to introduce it here. SAG: This is referring to the "cost of switching from no-prov to prov", which is low, and I'm only using this argument to explain why I look at system-level over the others. Performance overhead between system-level tools is a concern that I will address later on.]
+  they merely need to install some provenance tracer onto their system and run their code, without modifying it, in the tracer. ^[DSK: what about the performance penalty? Since you talk about performance in contributions, I think you have to introduce it here. SAG: This is referring to the "cost of switching from no-prov to prov", which is low, and I'm only using this argument to explain why I look at system-level over the others. Performance overhead between system-level tools is a concern that I will address later on. DSK: maybe add a word ("implementation"?) before cost to say which cost is meant here?]
 Perceived ease of use is a critical factor in the adoption of new technologies (formalized in the Technology Acceptance Model [@davisTechnologyAcceptanceModel1985]).
 Although the user may eventually use more semantically rich provenance, low-cost system-level provenance would get provenance's "foot in the door". 
 While this data is less rich than that of the workflow or application level, it may be enough to enable important applications such as reproducibility, caching, etc.
 Since system-level provenance collection is a possibly valuable tradeoff between cost and enabling provenance applications, system-level provenance will be the subject of this work.
 
-While there is little cost human-overhead in using system-level provenance (no user code change), there is a non-trivial implicit overhead in monitoring and recording each computational process.
-Even a minor overhead per I/O operation would get amplified over the tens of thousands of I/O operations that a program might execute per second.
+While there is little added human overhead in using system-level provenance (no user code change), there is a non-trivial implicit overhead in monitoring and recording each computational process.
+Even a minor overhead per I/O operation would be significant when amplified over the tens of thousands of I/O operations that a program might execute per second.
 
-Prior work in system-level provenance usually contains some benchmark programs to evaluate the overhead imposed by the system-level provenance tool.
```

Comment:
Should this really be "Prior work ... contain ..."

Reply:
it depends on if you think prior work is singular or plural. In this sense, it seems singular to me (one group), so I would say contains

Reply:
sure, seems fine